BUILD.tools

```diff
@@ -34,7 +35,7 @@ python_requirement(
 
 python_requirement(
     name="pytest-reqs",
-    resolve="pytest",
+    resolve="st2",
```

Comment:
This line (and some changes in `pants.toml`) moves the `pytest` versions from `lockfiles/pytest.lock` into `lockfiles/st2.lock`.